### PR TITLE
Eitan Ezra Hw #1

### PR DESCRIPTION
**Pull request by emezra**
I got the changes into README.md by forking the repository and making a clone of the file on my computer. I deleted the couple lines of code that were already in the file and wrote my biography. 
I thought that Markdown was fairly intuitive in terms of inserting emphases, lists, etc... after reading the R Markdown tutorial that was linked on the homework. One thing that I struggled with was getting the image in my biography to appear when I pushed the file to github, but realized that I had to push the image itself to github first for it to be able to access the image. Overall I really enjoyed the project, and thought that it was a good introduction to programming in R. I've only ever used R for statistical analyses, and didn't realize that I could use it to create something like a biography as well. I think that the GitHub workflow was a little bit more complicated to figure out than using R Markdown, but it started to make sense after experimenting a bit and reading the resources provided.

### CODE REVIEW

**Comment by emezra** [comment]
I was having a little bit of trouble with the homework submission, so the final submission is titled "New Final Submission." Hopefully everything works the way it should!

**Comment by mariekfp** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**       |      x     |              |            |
| **Presentation**           |        x   |              |            |
| **Achievement, creativity** |        x   |              |            |
| **Ease of access**          |    x       |              |            |

Remarks:

* Elaborate on above, especially for "needs work."
Eitan's work is clear and informative. I was easily able to both understand his code, and enjoy the finished bio page. 
* Some specific praise?
I liked the way you divided your bio up into sections and gave personal as well as professional information. 
* Something I learned?
Seeing how many commits you made, makes me realize I should probably commit more often (I only did 3 commits). 
* Specific constructive criticism?
In your description of your homework flow, it would've been helpful to include how you committed / pushed to GitHub. Did you use the GUI interface in R Studio or the terminal commands? 
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
Seems like you didn't really struggle with anything you couldn't figure out! I also struggled with the image to start with and figured out the issue in a similar way.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
-# hw01
-Repository for homework 01
+![Image of Me](GitHubPhoto.jpg)
+
+# Eitan's Homework #1: A Brief Biography
+
+My story begins in Brooklyn on April 2nd, 1998. I went to to Poly Prep Country Day School in Brooklyn where I was primarily engaged in high school debate. Now a Junior at the University of Chicago majoring in *Political Science* and *Psychology*.
```
Comment (bensoltoff) [inline README.md:5]: I need to stop looking at the birth years of my students. It reminds me I am old :(

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+My story begins in Brooklyn on April 2nd, 1998. I went to to Poly Prep Country Day School in Brooklyn where I was primarily engaged in high school debate. Now a Junior at the University of Chicago majoring in *Political Science* and *Psychology*. 
+
+### Career Goals
+
+I have changed my decision on what I would like to do several times in my college career. I started out sure that I wanted to go to law school, then flirted with the idea of consulting or working at a startup, and am now back to thinking I would like to go to law school at some point. My linkedin is attached [here](https://www.linkedin.com/in/eitan-ezra-35b53b165/) if you'd like a visual representation of what I've been doing with my time in the past. 
+
+### My Interests
+
+My life is not just school work. in my free time I like to:
+
```
Comment (bensoltoff) [inline README.md:14]: You don't need to assign the numbers yourself.

```
1. Read primarily about international relations.
1. Play basketball even though I'm still *very* bad.
1. Watch **mindless** Netflix shows about solving crimes.
```

renders as

1. Read primarily about international relations.
1. Play basketball even though I'm still *very* bad.
1. Watch **mindless** Netflix shows about solving crimes.

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |          x    |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |  x         |              |            |
| **Achievement, creativity** |    x      |              |            |
| **Ease of access**          |       x    |              |            |

**Pull request closed without merging**